internal/static/i18n/zh.yaml

```diff
@@ -0,0 +1,900 @@
+Errors:
+  Internal: 发生了内部错误
+  NoChangesFound: 没有变化
+  OriginNotAllowed: 这个"来源"是不被允许的
+  IDMissing: ID 丢失
+  ResourceOwnerMissing: 组织没有资源所有者
+  RemoveFailed: 无法移除
+  ProjectionName:
+    Invalid: 错误的映射名称
+  Assets:
+    EmptyKey: 资产的 Key 为空
```

Comment:
I didn't find the context of `Errors.Assets.EmptyKey` in the code, I don't have any more personal understanding of Key, I can only translate it literally, but in order to avoid ambiguity, I keep the original text of `Key` here.

Reply:
oh wow you are right. let's leaf it this way